### DISCUSSION TITLE
Dell Poweredge T420 and General Protection Fault (13) in UEFI pre-boot environment.

### DISCUSSION THREAD

**Discussion by stevensit-admin** [CloverHackyColor/CloverBootloader#684] [Q&A]
Hi,

I am probably doing something simple the wrong way, but I have got myself stuck.

I am trying to get Clover working with my Dell Poweredge T420. BIOS 2.9.0 with Linux as the OS, not that I think it is relevant with my issue. I have PERC H710's, running my SAS RAID, a PCIe M.2 NVME to PCIe card with a NVME drive installed.

The system works flawlessly booting my existing Ubuntu Server OS from my H710, with a RAID 10 and SAS Array.

As soon as I try and boot from a Clover USB, whether it is another USB or the otherwise working Hard drive I get the screenshot attached to the post. To be a little clearer. After I press enter to load from a device I get the screenshot. The clover GUI seems to load fine.

I tried to do some troubleshooting, created the USB using different methods, WIn11, and Linux, I tried different drives in different USB ports. I tried to add logging to the config.plist which didn't work, could be a part of the problem? I also tried a fresh install of clover after trying to play around with config's etc.

My goal is to eventually run my OS off the NVMe drive, but at this stage I can't even run the Live ISO through Clover.

I feel like I am missing something simple, but getting sick of hitting my head against a wall. 

Any help is appreciated.

Regards,
Paul.

![Screenshot 2024-04-29 175350](https://github.com/CloverHackyColor/CloverBootloader/assets/165717710/e41513a7-11ea-46cd-989a-2e6a284f59dd)

**Comment by Access-is-Denied** [comment]
Hi Paul,

I was stuck in a similar situation on my own T420, and was also smashing my head into a brick wall as a result, after a lot of time searching, I’ve finally found something that may help. It seems that the more recent versions of Clover don’t play very well with Dell Systems. Try using Clover v5125 or v5122, these seem to work. I’ve tried it on my own T420 and v5122 works for me, not getting these error messages anymore. I’ve linked the reddit thread I found below:

https://www.reddit.com/r/homelab/comments/1d7s8fy/dell_r720_nvme_boot_using_clover_general/

This is more of a workaround, but should at least get you up and running.

However, I’ve personally switched from using Clover to rEFInd since auto booting didn’t work for me on Clover no matter what I tried. rEFInd has worked perfectly for me after adding the NVME drivers, so I’ve left some links to installing rEFInd on a USB and adding the NVME drivers to it below in case you want to try that too:

https://forums.linuxmint.com/viewtopic.php?t=389479
 
https://unix.stackexchange.com/questions/765738/how-do-i-boot-from-an-nvme-disk-without-bios-support

Hope this helps :)

Kind Regards,
Ben

**Reply by DevR0m** [reply]
You just saved me a lot of time and headaches, thank you. I had the same issue as OP and using Clover v5122 worked perfectly when booting on my PCIe NVME.

**Reply by sitaramp** [reply]
Thanks @Access-is-Denied, I was having same issues with the R730. Clover v5122 worked as expected.
